src/components/ui/Text.stories.js

```diff
@@ -1,100 +1,197 @@
-// @flow
-
 import React from 'react'
 import { storiesOf } from '@storybook/react'
+import { text, boolean, select } from '@storybook/addon-knobs'
+import faker from 'faker'
 
 import Text from './Text'
 
+const StoryText = ({
+  heading,
+  subHeading,
+  headingType,
+  subHeadingType,
+  content,
+  isMarkdown,
+  isCentered,
+  hasDivider,
+  isPureContent,
+  required,
+  level
+}) => (
+  <Text
+    heading={text('heading', heading)}
+    headingType={select('headingType', typos, headingType)}
+    subHeading={text('subHeading', subHeading)}
+    subHeadingType={select('subHeadingType', typos, subHeadingType)}
+    level={text('level', level)}
+    content={text('content', content)}
+    isMarkdown={boolean('isMarkdown', isMarkdown)}
+    isCentered={boolean('isCentered', isCentered)}
+    hasDivider={boolean('hasDivider', hasDivider)}
+    required={boolean('required', required)}
+    isPureContent={boolean('isPureContent', isPureContent)}
+  />
+)
+
+const typos = {
+  badgeHeading: 'badgeHeading',
+  baseText: 'baseText',
+  divider: 'divider',
+  formText: 'formText',
+  heading: 'heading',
+  headline: 'headline',
+  labelText: 'labelText',
+  mainHeading: 'mainHeading',
+  sectionHeading: 'sectionHeading',
+  subheading: 'subheading',
```

Comment:
Linter has an error here. I suggest removing the comma
```suggestion
  subheading: 'subheading'
```